src/PrestaShopBundle/Bridge/AdminController/LegacyControllerBridge.php

```diff
@@ -23,35 +23,143 @@
+ * in the framework controller.
+ *
+ * @property int $id
+ * @property string $className
+ * @property string $controller_name
+ * @property string $php_self
+ * @property string $current_index
+ * @property string|null $position_identifier
+ * @property string $table
+ * @property string $token
+ * @property Employee $user
+ * @property array $meta_title
+ * @property array $breadcrumbs
+ * @property bool $lite_display
+ * @property string $display
+ * @property bool $show_page_header_toolbar
+ * @property string $page_header_toolbar_title
+ * @property array $toolbar_btn
+ * @property array $toolbar_title
+ * @property bool $display_header
+ * @property bool $display_header_javascript
+ * @property bool $display_footer
+ * @property bool $bootstrap
+ * @property array $css_files
+ * @property array $js_files
+ * @property string $tpl_folder
+ * @property array $errors
+ * @property array $warnings
+ * @property array $confirmations
+ * @property array $informations
+ * @property bool $json
+ * @property string $template
+ * @property array $tpl_vars
+ * @property array $modals
+ * @property int $multishop_context
+ * @property bool $multishop_context_group
  */
-trait SmartyTrait
+class LegacyControllerBridge implements LegacyControllerBridgeInterface
 {
     /**
-     * @param string $content
-     * @param Response|null $response
-     *
-     * @return Response
+     * @var ControllerConfiguration
      */
-    public function renderSmarty(string $content, Response $response = null, bool $isNewTheme = false): Response
-    {
-        $this->setMedia($isNewTheme);
+    private $controllerConfiguration;
+
+    /**
+     * @var FeatureInterface
+     */
+    private $multistoreFeature;
+
+    /**
+     * @var LegacyContext
+     */
+    private $legacyContext;
+
+    /**
+     * @var HookDispatcherInterface
+     */
+    private $hookDispatcher;
+
+    /**
+     * @var string[] maps legacy controller properties with the bridge
+     */
+    private $propertiesMap = [
+        'id' => 'controllerConfiguration.tabId',
+        'className' => 'controllerConfiguration.objectModelClassName',
+        'controller_name' => 'controllerConfiguration.legacyControllerName',
+        'php_self' => 'controllerConfiguration.legacyControllerName',
+        'current_index' => 'controllerConfiguration.legacyCurrentIndex',
+        'position_identifier' => 'controllerConfiguration.positionIdentifierKey',
+        'table' => 'controllerConfiguration.tableName',
+        'token' => 'controllerConfiguration.token',
+        'meta_title' => 'controllerConfiguration.metaTitle',
+        'breadcrumbs' => 'controllerConfiguration.breadcrumbs',
+        'lite_display' => 'controllerConfiguration.liteDisplay',
+        'display' => 'controllerConfiguration.displayType',
+        'show_page_header_toolbar' => 'controllerConfiguration.showPageHeaderToolbar',
+        'title' => 'controllerConfiguration.pageHeaderToolbarTitle',
```

Comment:
Identical to `page_header_toolbar_title` is it expected?

Reply:
good catch. Probably just a refacto mistake. Seems like property "title" doesn't exist (probably introduced by mistake during refacto), only the page_header_toolbar_title was expected here.

Reply:
it was added as an alias, but when I check `AdminController` it doesn't have this field indeed (nor does the parent `Controller` class)